packages/rxtranscribe/src/operators/toDeepSpeech.js

```diff
@@ -0,0 +1,137 @@
+// See: https://github.com/mozilla/DeepSpeech-examples/blob/r0.6/ffmpeg_vad_streaming/README.MD
```

Comment:
This module still needs some work but maybe worth a look if you're curious about how to use DeepSpeech models...  Basically, you download a DeepSpeech-compatible model (trained with Tensorflow), load the model into memory and then pipe audio chunks into it.